testclient.html

```diff
@@ -18,7 +18,15 @@
 				scriptEl.src = src.replace(/.*\/?data\//g, 'https://play.pokemonshowdown.com/data/');
 				document.head.appendChild(scriptEl);
 			}
-			var Config = {testclient: true};
+			var Config = {
+				testclient: true,
+				routes: {
+					root: 'pokemonshowdown.com',
+					client: 'play.pokemonshowdown.com',
+					dex: 'dex.pokemonshowdown.com',
+					replays: 'replay.pokemonshowdown.com'
+				},
+			};
```

Comment:
Maybe just call client config, and add `Config.testclient = true` after that line.